x-pack/plugins/data_visualizer/public/application/common/components/multi_select_picker/multi_select_picker.tsx

```diff
@@ -130,7 +137,15 @@ export const MultiSelectPicker: FC<{
             data-test-subj={`${dataTestSubj}-searchInput`}
           />
         </EuiPopoverTitle>
-        <div style={{ maxHeight: 250, overflow: 'auto' }}>
+        <div
+          css={
+            cssStyles?.filterItemContainer ??
+            css`
+              max-height: 250px;
```

Comment:
shall we introduce a variable for `250px`?

Reply:
Updated here [`2d35742` (#147137)](https://github.com/elastic/kibana/pull/147137/commits/2d35742e343066904b17034fa60a0b1c3fb5536e)